pkgs/games/betterspades/update.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl gnugrep gnused jq
+
+set -x -eu -o pipefail
+
+cd $(dirname "${BASH_SOURCE[0]}")
+
+get_last_commit(){
+    curl -sL "https://api.github.com/repos/$1/$2/commits/$3" | jq -r .sha
```

Comment:
This needs optionally to take a GITHUB_TOKEN especially if we do so many api calls or otherwise people get quickly rate limited